frontend/src/libs/components/meditation-tracks-list/meditation-track/meditation-track.tsx

```diff
@@ -0,0 +1,39 @@
+import { Icon, Link } from '#libs/components/components.js';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+
+import styles from './styles.module.scss';
+
+type Track = {
```

Comment:
I think you can place it into a separate file for consistency

Reply:
fix this comment